server/routes_show_test.go

```diff
@@ -0,0 +1,39 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/ollama/ollama/api"
+	"github.com/ollama/ollama/envconfig"
+	"github.com/ollama/ollama/llm"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShow(t *testing.T) {
```

Comment:
Why not put this in `routes_test.go`?

Reply:
In lieu of routes_create_test.go, routes_delete_test.go, routes_list_test.go, I thought show could be nice to separate out as well given the projector test. If we want to get rid of this file and just stick everything in routes_test, I'd change the create model handler to include model and projector KVs that can be retrieved in a show handler, but we might want to keep this stuff isolated. Lmk what you think

Reply:
Oh yeah and if you mean literally just sticking this function in routes_test.go, yeah its just to be consistent with the existence of those three other files Mike created